recipes/gdk-pixbuf/all/conanfile.py

```diff
@@ -0,0 +1,119 @@
+from conans import ConanFile, Meson, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import shutil
+
+
+class LibnameConan(ConanFile):
+    name = "gdk-pixbuf"
+    description = "toolkit for image loading and pixel buffer manipulation"
+    topics = ("conan", "gdk-pixbuf", "image")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://developer.gnome.org/gdk-pixbuf/"
+    license = "LGPL-2.1-or-later"
+    generators = "pkg_config"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_libpng": [True, False],
+        "with_libtiff": [True, False],
+        "with_libjpeg": ["libjpeg", "libjpeg-turbo", False],
+        "with_jasper": [True, False],
+        }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_libpng": True,
+        "with_libtiff": True,
+        "with_libjpeg": "libjpeg",
+        "with_jasper": False,
+        }
+
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+        if self.settings.os == "Macos":
+            # when running gdk-pixbuf-query-loaders
+            # dyld: malformed mach-o: load commands size (97560) > 32768
```

Comment:
I think I remember this error, it happened for me during GCC build. I'll try to find out what was the compiler flag to fix it.

Reply:
I'm wondering if dyld is not trying to load a static library